### ISSUE TITLE
Map should have non-path-based gesture to pan

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#5361]
As part of [WCAG success criterion 2.5.1](https://www.w3.org/WAI/WCAG22/Understanding/pointer-gestures), users need a way to complete all UI functionality without using a path-based gesture. Panning the map (swiping to change where you're looking) is considered a path-based gesture. We should add a way that users can move the map without swiping. This looks like it is most often accomplished with arrow keys, like in this example:

<img width="390" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3RZQ3c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e4a0d3807905e4e46bc07d138d3f486f4751b32f/Screenshot%202025-09-10%20at%2012.40.03%E2%80%AFPM.png" alt="Screenshot 2025-09-10 at 12.40.03 PM.png" />
